pantheon-files-daemon/FileManager1.vala

```diff
@@ -18,6 +18,7 @@
 
 [DBus (name = "org.freedesktop.FileManager1")]
 public class FileManager1 : Object {
+    const string reserved_chars = (GLib.Uri.RESERVED_CHARS_GENERIC_DELIMITERS + GLib.Uri.RESERVED_CHARS_SUBCOMPONENT_DELIMITERS);
```

Comment:
`reserved_chars` should be uppercase: `RESERVED_CHARS` for constants.

Reply:
modified to static string with replacements already in place.